src/core.js

```diff
@@ -240,6 +244,10 @@ function resumeProcessing( test ) {
 
 			begin();
 		}, 13 );
+
+		if ( test ) {
+			test.resuming = resume;
```

Comment:
Nope, because we don't expose the actual test object at any point in time (that I'm aware of at least).